dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala

```diff
@@ -1428,7 +1428,10 @@ private[tensor] class DenseTensor[@specialized(Float, Double) T: ClassTag](
   }
 
   override def reshape(sizes: Array[Int]): Tensor[T] = {
-    require(sizes.reduce(_ * _) == this.nElement())
+    require(sizes.reduce(_ * _) == this.nElement(),
+      "DenseTensor: nElement in sizes does not equal to that of objective Tensor," +
+        s" current input nElement in sizes is ${sizes.reduce(_ * _)}," +
+        s" objective Tensor nElement is ${this.nElement()}")
```

Comment:
"nElement of this tensor is not equal to nElement specified by sizes, sizes = ... and nElement of this tensor = ..."